### PR TITLE
added underline to text + why tech/values driven: computer component

### PR DESCRIPTION
**Pull request by dghazi12**
- Updated Computer component in Why Tech/Values Driven section with new SVG to include the lightbulb that was missing from the previous SVG
- Added underline to Why Tech/Values Driven headers and font size as per figma.
- Trello ticket [here](https://trello.com/c/9Q3kW3lP)

### CODE REVIEW

**Comment by gh0stl0nely** [comment]
@dghazi12 Nicely done! I am wondering if there is a way to increase the space between the underline and the word itself. If takes a long time figure out, we can merge this one first and put it up as an issue.

image vs 
image

File: src/components/WhyTechSVG/ValuesDriven.js [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
 
 const useStyles = makeStyles((theme) => ({ 
     cssHeaderText: {
         fontFamily: 'Josefin Sans',
         fontStyle: 'normal',
         fontWeight: '600',
-        fontSize: '36px',
-        lineHeight: '65px',
+        fontSize: '28px',
+        lineHeight: '55px',
```
Comment (hanlinc27) [inline src/components/WhyTechSVG/ValuesDriven.js]: line height gives me '44' in figma?

File: src/components/WhyTechSVG/WhyTech.js [hunk 2]
```diff
@@ -7,17 +7,19 @@ const useStyles = makeStyles((theme) => ({
         fontFamily: 'Josefin Sans',
         fontStyle: 'normal',
         fontWeight: '600',
-        fontSize: '36px',
-        lineHeight: '57px',
+        fontSize: '28px',
+        lineHeight: '65px',
```
Comment (hanlinc27) [inline src/components/WhyTechSVG/WhyTech.js]: same comment as above ^

File: src/components/WhyTechSVG/WhyTech.js [hunk 3]
```diff
@@ -7,17 +7,19 @@ const useStyles = makeStyles((theme) => ({
...
         color: '#000000',
         top: '856px',
+        textDecoration: 'underline',
+        textDecorationColor: '#FF7500',
     },
     cssBodyText: {
         fontFamily: 'poppins',
         fontStyle: 'normal',
-        fontWeight: 'normal',
+        fontWeight: '400',
```
Comment (hanlinc27) [inline src/components/WhyTechSVG/WhyTech.js:21]: figma says font-weight should be '600', is there a reason it's set to 400?

File: src/components/WhyTechSVG/ValuesDriven.js [hunk 4]
```diff
@@ -1,22 +1,23 @@
...
         alignItems: 'center',
         color: '#000000',
+        textDecoration: 'underline',
+        textDecorationColor: '#FF7500',
     },
     cssBodyText: {
         fontFamily: 'poppins',
         fontStyle: 'normal',
-        fontWeight: 'normal',
+        fontWeight: '400',
```
Comment (hanlinc27) [inline src/components/WhyTechSVG/ValuesDriven.js:18]: font-weight is also shown as 600 in figma

**Review by hanlinc27 — CHANGES_REQUESTED** [review]

**Review by harshvchawla — COMMENTED** [review]
nothing to add; I guess can be merged post Hanlin's observations are taken care of

**Comment by dghazi12** [comment]
> @dghazi12 Nicely done! I am wondering if there is a way to increase the space between the underline and the word itself. If takes a long time figure out, we can merge this one first and put it up as an issue.
> 
> image vs
> image

@gh0stl0nely Yea initially I read that it was a font issue that was causing the break up. Found a different approach now however and I will try to implement it.

File: src/components/WhyTechSVG/ValuesDriven.js (hunk 1 above)
Comment (dghazi12) [reply src/components/WhyTechSVG/ValuesDriven.js]: @hanlinc27 Yea the line height when I set it to 44px seemed off on my end compared to figma. My goal for this pr was just to update the computer image but you're right I should just fix this issue one time.

File: src/components/WhyTechSVG/WhyTech.js (hunk 3 above)
Comment (dghazi12) [reply src/components/WhyTechSVG/WhyTech.js:21]: @hanlinc27 As for the font weight, not sure where you are seeing that. I see 600 for the header text and 400 for the body text on figma?

File: src/components/WhyTechSVG/ValuesDriven.js (hunk 1 above)
Comment (dghazi12) [reply src/components/WhyTechSVG/ValuesDriven.js]: @hanlinc27 I fixed the line height for both to match figma.

**Comment by dghazi12** [comment]
> @dghazi12 Nicely done! I am wondering if there is a way to increase the space between the underline and the word itself. If takes a long time figure out, we can merge this one first and put it up as an issue.
> 
> image vs
> image

@gh0stl0nely All fixed now. I used border to create the line and had to add another foreignObject tag to be able to manipulate the 'underline' to match figma. Seems to be the only way around this without having to change the font.

**Review by gh0stl0nely — APPROVED** [review]
@dghazi12 Very interesting solution indeed! It seems there is another solution with `text-underline-position: under;` on StackOverFlow but that isn't cross-browser safe. I am good with this. If Hanlin's request is also fixed then we can merge.

**Review by demiwu96 — APPROVED** [review]

**Pull request merged**